app.rb

```diff
@@ -0,0 +1,73 @@
+require_relative 'lib/person/student'
+require_relative 'lib/person/teacher'
+require_relative 'lib/book'
+require_relative 'lib/rental'
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    if @books.empty?
+      print 'There are no registered books'
+    else
+      print 'Books:'
+      @books.each do |book|
+        print "Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+  end
+
+  def list_people
+    if @people.empty?
+      print 'There are no registered people'
+    else
+      print 'People:'
+      @people.each do |person|
+        print "Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+  end
+
+  def create_teacher(age, specialization, name)
+    teacher = Teacher.new(age, specialization, name)
+    @people << teacher
+    puts 'Teacher created successfully'
+  end
+
+  def create_student(age, classroom, name, option)
+    permission = false
+    permission = true if %w[y Y].include?(option)
+    student = Student.new(age, classroom, name, permission)
+    @people << student
+    puts 'Student created successfully'
+  end
+
+  def create_book(title, author)
+    book = Book.new(title, author)
+    @books << book
+    puts 'Book created successfully'
+  end
+
+  def create_rental(date, book, person)
+    rental = Rental.new(date, book, person)
+    @rentals << rental
+    puts 'Rental created successfully'
+  end
```

Comment:
- [ ] Great work on providing all the options for adding new users, books, and rentals :+1:. However, as evidenced below, every time I attempted to add any of them, the application displayed an error. This could be due to a potential issue with the correct syntax for each parameter.

![Screenshot from 2023-10-20 18-12-00](https://github.com/BiancaMe/school-library/assets/105117832/8f5cfbb7-d927-457f-bff2-2300195c3197)

- [ ] This is happening also with `List all rentals for a given person id`